docs/source/tips.rst

```diff
@@ -35,15 +34,95 @@ vecLib is the default BLAS library installed on Mac OS X.
 We recommend using other BLAS libraries such as `OpenBLAS <http://www.openblas.net/>`_.
 
 To use an alternative BLAS library, it is necessary to reinstall NumPy.
-Here is an instruction to install NumPy with OpenBLAS using `Homebrew <https://brew.sh/>`_.
+Here are instructions to install NumPy with OpenBLAS using `Conda <https://conda.io/docs/user-guide/install/index.html>`_.
+
+::
+
+   $ conda install -c conda-forge numpy
+
+Otherwise, to install NumPy without Conda, you may need to install NumPy from source.
+
+Use `Homebrew <https://brew.sh/>`_ to install OpenBLAS.
 
 ::
 
-   $ brew tap homebrew/science
    $ brew install openblas
-   $ brew install numpy --with-openblas
 
-If you want to install NumPy with pip, use `site.cfg <https://github.com/numpy/numpy/blob/master/site.cfg.example>`_ file.
+Uninstall existing NumPy installation
+
+::
+
+   $ pip uninstall numpy
+
+Install NumPy from the source code
+
+::
+
+   $ git clone https://github.com/numpy/numpy
+   $ cd numpy
+   $ git checkout refs/tags/v1.13.3
+   $ cp site.cfg.example site.cfg
+
+You'll need to edit the site.cfg file around line 128 to set the library to OpenBLAS by removing the comment #s.
+
+::
+
+   [openblas]
+   libraries = openblas
+   library_dirs = /usr/local/opt/openblas/lib
+   include_dirs = /usr/local/opt/openblas/include
+   runtime_library_dirs = /usr/local/opt/openblas/lib
+
+Check the setup config file.
+
+::
+
+   $ python setup.py config
+
+If you see these settings in the log, it should be ok:
+
+::
+
+    openblas_info:
+      FOUND:
+        libraries = ['openblas', 'openblas']
+        library_dirs = ['/usr/local/opt/openblas/lib']
+        language = c
+        define_macros = [('HAVE_CBLAS', None)]
+        runtime_library_dirs = ['/usr/local/opt/openblas/lib']
+
+Continue with the build. This takes a few minutes.
+
+::
+
+   $ python setup.py build
```

Comment:
It could be mentioned that `python setup.py clean` might be required to retry installing from the source.